app/src/main/res/resource/values-de/strings.xml

```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
     <string name="about_unsplash">Freie (mach, was immer du willst) hochauflösende Fotos.</string>
```

Comment:
Mhh, @WangDaYeeeeee, does "free" in this slogan really refer to freedom or to gratis ("free as in beer")? I mean the pictures are obviously both, but for a slogan maybe "kostenlos" or "gratis" sound better…